kie-drools-wb/kie-drools-wb-distribution-wars-cdi1.0/pom.xml

```diff
@@ -343,6 +343,11 @@
       <groupId>org.jbpm</groupId>
       <artifactId>jbpm-services-ejb-timer</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.woodstox</groupId>
+      <artifactId>stax2-api</artifactId>
+      <version>3.1.1</version>
+    </dependency>
```

Comment:
I think it should be possible to include `dependencyManagement` section directly in `kie-wb-distribution-wars-cdi1.0` module and define (override) the version there. Then the `dependencies` section can use the artifact without specifying version and it will get the correct one. Then we should also be able to re-enable the enforcer again.

I will look into this and submit a PR in case I make it work.